### ISSUE TITLE
issue with loop

### ISSUE THREAD

**Issue by czarczar** [goldfire/howler.js#435]
if you first loop a sound and during playing:
- pause
- stop
- change volume
etc.
Those actions are only submitted to the part you are listening to, not to the next time in the loop.
I suppose this is due to the fact that the next loop is set-up at the start of playing the current loop.
(tested under: Chrome Version 46.0.2490.71 (64-bit) / Linux Mint).

**Comment by goldfire** [comment]
What version are you using?

**Comment by czarczar** [comment]
I use version 1.1.29
I found a work-around for my situation. My situation being: I have a looping sound playing, which should be paused when another sound starts playing and continue when the other sound ends. Pause does not stop the looping sound totally, but if I pause, set volume to zero and stop looping it works fine.
So for the second sound I created onplay and onend functions that pause and continue the looping sound:
sounds[1]=new Howl({
	urls: ['sound1.mp3'],
	autoplay: true,
	loop: true
});
sounds[2]=new Howl({
	urls: ['sound2.mp3'],
	autoplay: false,
	loop: false,
	onplay:function () {sounds[1].pause(); sounds[1].volume(0); sounds[1].loop(false)},
	onend:function () {sounds[1].loop(true); sounds[1].volume(1); sounds[1].play()}
});
Thanx for Howler, I love it :-)
It would be nice if I could find a way to check when the sounds are really loaded (I have 183 sounds preloading, in total about 5.2 MB).
